### ISSUE TITLE
Add automatic accessibility tests to pipeline

### ISSUE THREAD

**Issue by thrbnhrtmnn** [deven-org/boiler#565]
#### Description / User Story

With this task we want add automatic accessibility testing to our pipeline. In the future we want to make these tests also prevent a merge, if the tests fail, but before we can do that we probably have to fix the issues that already are in the components. So for now the tests should not prevent a merge.

To prepare the follow up task #595 , this task also includes the documentation and prioritisation of all the errors we receive once the automatic tests has been running for all components.

----
#### Acceptance Criteria
- [ ] Automatic accessibility tests have been added to the pipeline
- [ ] Automatic accessibility test has been running for all components that will be part of alpha release (TBD):
   - [ ] TextButton
   - [ ] IconButton
   - [ ] Checkbox
   - [ ] NumberInput
   - [ ] Radio
   - [ ] RadioGroup
   - [ ] TextArea
   - [ ] TextInput
   - [ ] Select
   - [ ] ToggleSwitch
   - [ ] Counter
   - [ ] FormCaption
   - [ ] FormLabel
   - [ ] ButtonGroup
   - [ ] Divider
   - [ ] Loader
   - [ ] Tooltip  
- [ ] All issues that were found have been documented and prioritised

----
#### Background information
- ...
